i18n/pt/campaigns/eoe/ice_and_death_part_2.po

```diff
@@ -12,114 +12,116 @@ msgstr ""
 
 msgid "Each investigator's base starting resources is equal to the shelter value of the camp, instead of five."
-msgstr ""
+msgstr "Os recursos iniciais básicos de cada investigador são iguais ao valor do abrigo do acampamento, em vez de cinco."
 
 msgid "Each investigator may choose one available member of the expedition team who is not crossed off or marked \"MIA\" to join them. Put the story asset for the chosen character into play in that investigator's play area. Place damage and horror on that character as indicated in the \"Expedition Team\" section of the Campaign Log."
-msgstr ""
+msgstr "Cada investigador pode escolher um membro disponível da equipe de expedição que não esteja riscado ou marcado como \"MIA\" para se juntar a eles. Coloque o recurso da história para o personagem escolhido em jogo na área de jogo desse investigador. Coloque dano e horror em esse personagem conforme indicado na seção \"Equipe de Expedição\" do Registro de Campanha."
 
 msgid "Scenario Steup (from Checkpoint)"
-msgstr ""
+msgstr "Configuração do cenário (do checkpoint)"
 
 msgid "Search the encounter deck and all out-of-play areas for all cards from <i>The Crash</i> and <i>Creatures in the Ice</i> encounter sets. Remove each card from these encounter sets from the game."
-msgstr ""
+msgstr "Procure no deck de encontro e em todas as áreas fora de jogo por todos os cards dos conjuntos de encontro <i>O acidente</i> e <i>Creaturas no Gelo</i>. Remova cada carta desses conjuntos de encontro do jogo."
 
 msgid "Build the agenda deck using agendas 4-6 and the act deck using act 2. (These cards can be found in the <i>Lost in the Night</i> encounter set."
-msgstr ""
+msgstr "Construa o deck de agendas usando as agendas 4-6 e o ​​deck de atos usando o ato 2. (Estas cartas podem ser encontradas no conjunto de encontros <i>Perdidos na Noite</i>."
 
 msgid "Put each remaining set-aside [[Uncharted]] location into play."
-msgstr ""
+msgstr "Coloque cada local reservado [[Uncharted]] restante em jogo."
 
 msgid "Place clues on each revealed location equal to its clue value."
-msgstr ""
+msgstr "Coloque pistas em cada local revelado igual ao seu valor de pista."
 
 msgid "Several members of the expedition go missing in the night!"
-msgstr ""
+msgstr "Vários membros da expedição desaparecem durante a noite!"
 
 msgid "Find each [[Possessed]] enemy from the <i>Lost in the Night</i> encounter set whose title matches one of the Expedition Team members marked \"MIA\" in your Campaign Log. Set each of these enemies aside, facedown. Remove the rest from the game."
-msgstr ""
+msgstr "Encontre cada [[Possessed]] inimigo do conjunto de encontros <i>Perdidos na Noite</i> cujo título corresponda a um dos membros da Equipe de Expedição marcado como \"MIA\" em seu Registro de Campanha. Defina cada um desses inimigos lado, virado para baixo. Remova o resto do jogo."
 
 msgid "#name# is MIA."
```

Comment:
So this line is a placeholder that will be substituted with character names to indicate each one has gone missing in the log.